### PR TITLE
fix issue for #264, respect custom tmp dir for options given to constructor

### PR DESCRIPTION
**Pull request by esler**
see #264

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
         "mikehaertl/php-tmpfile": "^1.0.0",
         "mikehaertl/php-shellcommand": "^1.0.2"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^6.0"
+    },
```
Comment (mikehaertl) [inline composer.json]: Please remove this. We don't want every user to get phpunit installed with this package by default.

File: src/Pdf.php [hunk 2]
```diff
@@ -262,6 +262,16 @@ public function getPdfFilename()
...
 
+    /**
+     * Returns an options
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->_options;
+    }
```
Comment (mikehaertl) [inline src/Pdf.php]: I'm not sure, if I want to expose this. Can you give a use case where someone would ever need the content of the internal `_options` array?

**Comment by mikehaertl** [comment]
@esler I hope you don't mind, but I will take car of the fix and adding the test for it. Thanks anyway for finding this bug.

File: src/Pdf.php (hunk 2 above)
Comment (esler) [reply src/Pdf.php]: for testing purposes - it's definitely valid

File: composer.json (hunk 1 above)
Comment (esler) [reply composer.json]: ok, no prob... but it's good practice. It's in the "require-dev" so if user don't want it, he can do
```
composer install --no-dev
```

**Comment by esler** [comment]
ok, no prob, you did no harm

**Pull request closed without merging**